.circleci/config.yml

```diff
@@ -232,6 +232,14 @@ jobs:
             for prof in $(ls /tmp/workspace/profiles/); do
               tail -n +2 /tmp/workspace/profiles/"$prof" >> coverage.txt
             done
+      - run:
+          name: filter out DONTCOVER
```

Comment:
I am not sure hiding files from coverage stats is a great idea.
Unless, it is a very clear case, it is good to be upfront wrt coverage values.

Reply:
Cobra commands cannot be easily tested. That's why we have the `cli_test` test suite.